### DISCUSSION TITLE
Got a virus warning?

### DISCUSSION THREAD

**Discussion by hakusaro** [Pryaxis/TShock#2369] [Announcements]
If you got a virus warning when downloading TShock (from something like Malwarebytes Antimalware), this is a false positive. TShock isn’t a virus or ransomware. You can check what other antivirus engines detect using [VirusTotal](https://www.virustotal.com/gui/). A [scan of 4.5.4](https://www.virustotal.com/gui/file/a745c51b12762260d7b0865f3a6c708a3e08056065e7431b5183d134ff1328a9/detection) didn’t even get flagged by MBAM.

If you get a notification like this about TShock, please contact your antivirus vendor and report that they have a false positive. **I have already tried to get a problem with false positives fixed with Malwarebytes in the past as part of my job, and it hasn’t actually gotten anywhere. They only listen to customers.**

If you want, you can build TShock yourself, to avoid downloading software from us. If you need a list of reasons why you should trust me, well, I don’t have one available. But antivirus engines are definitely wrong.

Sorry for the confusion!

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/70816842/121015706-f3dee580-c7a3-11eb-8118-57f186d203f8.png)

**Reply by hakusaro** [reply]
What is the purpose of this post?

**Reply by ghost** [reply]
none
